src/modules/subscribers.ts

```diff
@@ -0,0 +1,36 @@
+import { EntitySubscriberInterface, EventSubscriber, InsertEvent, LoadEvent, UpdateEvent } from "typeorm"
+
+function updateNulls(entity:any) {
+    const that: any = entity;
+    Object.keys(entity).forEach((k) => {
+        if (that[k] === null) that[k] = undefined;
+    });
+  }    
+
+@EventSubscriber()
+export class NullCheckerSubscriber<T> implements EntitySubscriberInterface<T> {
+    constructor(private cls: new (...a:any) => T) {
+
+    }
+    listenTo(): any {
+      return this.cls;
+    }
```

Comment:
Don't include this method. It says if you want to subscribe to all entities to *not* define this. 

> To listen to any entity you just omit `listenTo` method and use `any`

https://github.com/typeorm/typeorm/blob/master/docs/listeners-and-subscribers.md